automation/taskcluster/lib/tasks.py

```diff
@@ -436,6 +452,115 @@ def craft_push_task(
             },
         )
 
+    def craft_raptor_tp6m_cold_task(self, for_suite):
+
+        def craft_function(signing_task_id, mozharness_task_id, variant, gecko_revision, force_run_on_64_bit_device=False):
```

Comment:
We never call `_craft_raptor_task` directly.
Perhaps instead of defining that function, then making a closure that wraps it with default values, why don't we just move the defaults into `_craft_raptor_task` itself and remove the need for `craft_raptor_tp6m_cold_task`?

Perhaps this is similar to my argument about `printGeckoviewVersions` - it seems like we're adding complexity ahead-of-time before it's warranted

Reply:
`_craft_raptor_task` is going to be called real soon, when new job types are enabled (like they are in r-b). I'd prefer this function around. This will make job addition easier.